composer.json

```diff
@@ -16,19 +16,20 @@
         }
     ],
     "minimum-stability": "dev",
+    "prefer-stable": true,
     "require": {
         "php": ">=5.3.9",
-        "symfony/framework-bundle": "~2.3.27 || ~2.6.6 || ~2.7 || ~3.0",
+        "symfony/framework-bundle": "~2.3.27 || ^2.6.6 || ~3.0",
         "symfony/doctrine-bridge": "~2.3 || ~3.0",
-        "phpcr/phpcr-implementation": "2.1.*",
-        "phpcr/phpcr-utils": "~1.2.7 || ~1.3"
+        "phpcr/phpcr-implementation": "~2.1",
+        "phpcr/phpcr-utils": "^1.2.7"
     },
     "require-dev": {
-        "phpunit/phpunit": "~4.5@stable",
+        "phpunit/phpunit": "~4.5",
         "phpunit/php-code-coverage": "@stable",
         "symfony/form": "~2.3 || ~3.0",
-        "doctrine/phpcr-odm": "~1.3@dev",
-        "symfony-cmf/testing": "~1.2,>=1.2.7",
+        "doctrine/phpcr-odm": "~1.3",
+        "symfony-cmf/testing": "^1.2.7",
```

Comment:
@Soullivaneuh you need to adress this one to hopefully get travis to build again